tempto-core/src/test/groovy/io/trino/tempto/internal/configuration/YamlConfigurationTest.groovy

```diff
@@ -60,10 +60,37 @@ list:
     def createFromAllComments()
     {
         setup:
-        def configuration = new YamlConfiguration("# comment");
+        def configuration = new YamlConfiguration("# comment")
 
         expect:
         configuration.listKeys() == [] as Set
         configuration.getString('missing') == Optional.empty()
     }
+
+    def createWithCopyFromAnchor()
+    {
+        setup:
+        def configuration = new YamlConfiguration("""\
+root: &root
+  one:
+    q: life, universe and everything
+    a: 42
+  two:
+    - quick
+    - brown
+    - fox
+
+copy: 
+  <<: *root
```

Comment:
Can we also have a test where we override keys from the anchor?

e.g. changing `one.q` to something else in under `copy`?